app/assets/javascripts/application.js

```diff
@@ -18,14 +18,24 @@
 $(document).ready(function(){
   $(".dropdown-button").dropdown();
   $(".button-collapse").sideNav();
+  $('.parallax').parallax();
+  $('select').material_select();
```

Comment:
Mixed double and single quotes.<br>Identifier 'material_select' is not in camel case.